apps/marketing/src/app/(marketing)/open/page.tsx

```diff
@@ -43,18 +47,28 @@ export type StargazersType = z.infer<typeof ZStargazersLiveResponse>;
 export type EarlyAdoptersType = z.infer<typeof ZEarlyAdoptersResponse>;
 
 export default async function OpenPage() {
-  const {
-    forks_count: forksCount,
-    open_issues: openIssues,
-    stargazers_count: stargazersCount,
-  } = await fetch('https://api.github.com/repos/documenso/documenso', {
-    headers: {
-      accept: 'application/vnd.github.v3+json',
+  const { forks_count: forksCount, stargazers_count: stargazersCount } = await fetch(
```

Comment:
Is there a reason we aren't doing all these fetch requests in parallel?

https://nextjs.org/docs/app/building-your-application/data-fetching/patterns#parallel-data-fetching

Reply:
I've updated it to use parallel routes